cli/tests/integration_tests.rs

```diff
@@ -9,6 +9,7 @@ use test_util as util;
 
 #[test]
 fn std_tests() {
+  std::env::set_var("TZ", "UTC");
```

Comment:
This sets `TZ` env variable in the current env - that's not desirable, instead use `.env()` on the subprocess like for `DENO_DIR` below